tests/hr/_fixtures.yaml

```diff
@@ -1,20 +1,45 @@
 language: hr
 areas:
```

Comment:
I think we need nominativ, dativ and acusativ for all examples below. Can you please fix it accordingly so it's consistent?
Nominativ for when we just want to name a room where something should be done
Dativ for the "ugasi ... u {{ area }}" example
Acusatov for the "ugasi {{ area }}" example

Reply:
looks good